internal/pkg/aws/resourcegroups/resourcegroups.go

```diff
@@ -32,69 +27,63 @@ type tagFilter struct {
 	Values []string
 }
 
-type query struct {
-	ResourceTypeFilters []string
-	TagFilters          []tagFilter
+// Resource contains the ARN and the tags of the resource.
+type Resource struct {
+	Arn  string
```

Comment:
nit: 
```suggestion
	ARN  string
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms